Tests/SwiftIfConfigTest/VisitorTests.swift

```diff
@@ -0,0 +1,183 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import XCTest
+import SwiftSyntax
+import SwiftParser
+import SwiftIfConfig
+import _SwiftSyntaxTestSupport
+
+/// Visitor that ensures that all of the nodes we visit are active.
+///
+/// This cross-checks the visitor itself with the `SyntaxProtocol.isActive(in:)`
+/// API.
+class AllActiveVisitor: ActiveSyntaxAnyVisitor<TestingBuildConfiguration> {
+  init(configuration: TestingBuildConfiguration) {
+    super.init(viewMode: .sourceAccurate, configuration: configuration)
+  }
+  open override func visitAny(_ node: Syntax) -> SyntaxVisitorContinueKind {
+    XCTAssertTrue(try! node.isActive(in: configuration))
```

Comment:
Want to wrap this in `XCTAssertNoThrows` instead of `try!`?

Reply:
Sure